app/assets/stylesheets/nucore-overrides.scss

```diff
@@ -1,4 +1,7 @@
 /*
+UCONN Fork Styling
```

Comment:
I think we decided on including this stuff directly in the existing CSS instead of overriding what exists; I don't think this should be in an overrides file.

Reply:
Can we get rid of this comment?